src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,43 +1,39 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.storage.IntLinkedList;
+import edu.kis.vh.nursery.storage.IntStorageInterface;
 
 public class DefaultCountingOutRhymer {
 
-	IntArrayStack intArrayStack = new IntArrayStack();
-	IntLinkedList intLinkedList = new IntLinkedList();
+	private IntStorageInterface intStorageInterface;

```

Comment:
"interface" suffix of the attribute name is wrong and does not show the meaning.